storage/rocksdb/clone/donor.cc

```diff
@@ -68,7 +68,9 @@ class [[nodiscard]] rdb_checkpoint final {
   // Returns MySQL error code
   [[nodiscard]] int init() {
     assert(!m_active);
-    const auto result = myrocks::rocksdb_create_checkpoint(m_dir.c_str());
+    m_full_dir = make_dir_full_name(
```

Comment:
It might be confusing to have `m_dir` and `m_full_dir` and `make_dir_name` and `make_dir_full_name`. After this patch, the dir name is the one that has all the suffixes, and the previous dir name is a prefix. Please rename `make_dir_name` to `make_dir_prefix_name`, `make_dir_full_name` to `make_dir_name`, and member variables too.

Reply:
resolved